dotnet-core/dotnet-core/WeatherForecast.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+
+namespace dotnet_core
+{
+    public class WeatherForecast
+    {
+        public DateTime Date { get; set; }
+
+        public int TemperatureC { get; set; }
+
+        public int TemperatureF => 32 + (int)(TemperatureC / 0.5556);
+
+        public string Summary { get; set; }
+    }
```

Comment:
Bu nedir abi anlamadım bu class ne için

Reply:
.net core'un kendisi olusturdugu bir şey

Reply:
forcast ne olum dasdsa

Reply:
knk .net de proje olustururken WeatherForecast.cs ve WeatherForecastController.cs diye iki ornek olusturuyor basit bir API örnek göstermek için askdajlsdj